recipes/octomap/byte.patch

```diff
@@ -0,0 +1,37 @@
+diff --git a/octomap/src/binvox2bt.cpp b/octomap/src/binvox2bt.cpp
```

Comment:
This patch is needed because `octomap` does not seem to be compatible with C++17. This is the error I get with the compiler from conda (`x86_64-conda_cos6-linux-gnu-c++ (crosstool-NG fa8859cb) 7.2.0`):
<details>

```
/tmp/octomap/octomap/src/binvox2bt.cpp: In function 'int main(int, char**)':
/tmp/octomap/octomap/src/binvox2bt.cpp:230:9: error: reference to 'byte' is ambiguous
         byte value;
         ^~~~
/tmp/octomap/octomap/src/binvox2bt.cpp:59:23: note: candidates are: typedef unsigned char byte
 typedef unsigned char byte;
                       ^~~~
In file included from /tmp/octomap/octomap/include/octomap/AbstractOcTree.h:37:0,
                 from /tmp/octomap/octomap/include/octomap/AbstractOccupancyOcTree.h:37,
                 from /tmp/octomap/octomap/include/octomap/OccupancyOcTreeBase.h:45,
                 from /tmp/octomap/octomap/include/octomap/OcTree.h:38,
                 from /tmp/octomap/octomap/include/octomap/octomap.h:37,
                 from /tmp/octomap/octomap/src/binvox2bt.cpp:52:
/home/local/lesteve/miniconda3/envs/robotics/x86_64-conda_cos6-linux-gnu/include/c++/7.3.0/cstddef:66:14: note:                 enum class std::byte
   enum class byte : unsigned char {};
              ^~~~
/tmp/octomap/octomap/src/binvox2bt.cpp:231:9: error: reference to 'byte' is ambiguous
         byte count;
         ^~~~
/tmp/octomap/octomap/src/binvox2bt.cpp:59:23: note: candidates are: typedef unsigned char byte
 typedef unsigned char byte;
                       ^~~~
In file included from /tmp/octomap/octomap/include/octomap/AbstractOcTree.h:37:0,
                 from /tmp/octomap/octomap/include/octomap/AbstractOccupancyOcTree.h:37,
                 from /tmp/octomap/octomap/include/octomap/OccupancyOcTreeBase.h:45,
                 from /tmp/octomap/octomap/include/octomap/OcTree.h:38,
                 from /tmp/octomap/octomap/include/octomap/octomap.h:37,
                 from /tmp/octomap/octomap/src/binvox2bt.cpp:52:
/home/local/lesteve/miniconda3/envs/robotics/x86_64-conda_cos6-linux-gnu/include/c++/7.3.0/cstddef:66:14: note:                 enum class std::byte
   enum class byte : unsigned char {};
              ^~~~

```

</details>